Update load-balancer-multiple-ip.md

ShuheiUda opened the pull request:
Please add step2.
Without IP settings inside VMs, probe will fail.
https://github.com/MicrosoftDocs/azure-docs/blob/master/includes/virtual-network-multiple-ip-addresses-os-config.md

azurecla commented:
Hi __@Juliako__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (Juliako). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ShuheiUda committed:
Update load-balancer-multiple-ip.md

Please add step2.
Without IP settings inside VMs, probe will fail.
https://github.com/MicrosoftDocs/azure-docs/blob/master/includes/virtual-network-multiple-ip-addresses-os-config.md
